posthog/api/insight.py

```diff
@@ -199,6 +216,7 @@ class InsightSerializer(InsightBasicSerializer):
     A dashboard tile ID and dashboard_id for each of the dashboards that this insight is displayed on.
     """,
     )
+    query = serializers.JSONField(required=False, allow_null=True, help_text="Query node JSON string")
```

Comment:
I echoed the existing API docs text here.We should follow-up with nicer docs for this...